src/engraving/libmscore/cmd.cpp

```diff
@@ -655,9 +655,12 @@ void Score::expandVoice()
 
 void Score::addInterval(int val, const std::vector<Note*>& nl)
 {
+    std::vector<Note*> notes;
```

Comment:
Even though the existing code is not really doing a great job at this, it would be good to give the variables slightly clearer names. For example, I would rename this one to `newNotes`, and you could rename `nl` to `notes`. Generally, in this method, it would be nice to consistently call the existing notes `oldNote` and the newly created ones `newNote`.